### DISCUSSION TITLE
Access denied when trying to switch to different version

### DISCUSSION THREAD

**Discussion by dMaisnam** [coreybutler/nvm-windows#735] [Support] [answered]
Hi. I currently have node `14.6.0`. I tried to install `nvm-windows` following an article. When I completed the setup process, both my `nvm -v` and `node -v` commands were not working.

EDIT: I had to restart the shell session to make it work. Now, I am encountering a new issue. Where when I have successfully installed another node version using example `nvm install 16`, and try to use the new version using `nvm use 16.x.x`, it gives me an `exit status  5: Access denied`. How do I solve this?

Thanks for your time.

**Comment by coreybutler** [comment, marked as answer]
Please see the [common issues wiki](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#permissions-exit-1-exit-5-access-denied-exit-145).
